adserver/views.py

```diff
@@ -722,6 +734,35 @@ def get_response(self, request, advertisement, publisher):
         return HttpResponseRedirect(url)
 
 
+class AdViewTimeProxyView(AdViewProxyView):
```

Comment:
What are we using from the `AdViewProxyView` that requires subclassing? Seems we're removing most of the logic?

Reply:
Mostly just getting the relevant offer, publisher, and ad.